apps/web/src/pages/studio-onboarding/components/Wrapper.tsx

```diff
@@ -1,14 +1,19 @@
 import { css, cx } from '@novu/novui/css';
-import { ReactNode } from 'react';
+import { ReactNode, useEffect } from 'react';
 
 export const Wrapper = ({ children, className }: { children: ReactNode; className?: string }) => {
+  useEffect(() => {
+    document.body.setAttribute('style', 'overflow: auto');
+  }, []);
+
   return (
     <div
       className={cx(
         css({
           width: '100dvw',
-          height: '100dvh',
+          minHeight: '100dvh',
           colorPalette: 'mode.cloud',
+          paddingBottom: '4rem',
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** If we're going to use a non-DS value, please make into a semantic token so we only change it in one place‏